src/itest/resources/features/company-insolvency-delete.feature

```diff
@@ -17,3 +17,28 @@ Feature: Delete company insolvency information
     Then I should receive 404 status code
     And the CHS Kafka API is not invoked
 
+  Scenario: Processing delete company insolvency without 'x-request-id' key in the header
+
+    Given Insolvency data api service is running
+    When I send DELETE request without x-request-id key in the header
+    Then I should receive 500 status code
+    And the CHS Kafka API is not invoked
+
+  Scenario: Processing delete company insolvency while database is down
+
+    Given Insolvency data api service is running
+    And the insolvency information exists for "CH3634545"
+    And the insolvency database is down
+    When I send DELETE request with company number "CH3634545"
+    Then I should receive 503 status code
+    And the CHS Kafka API is not invoked
```

Comment:
One or two scenarios mentioned in the ticket not implemented it seems, like entry not exists in DB, bad request from chs-data-api etc?